config/default/core.extension.yml

```diff
@@ -141,6 +140,7 @@ module:
   menu_link_content: 0
   menu_ui: 0
   metatag: 0
+  monolog: 0
```

Comment:
Likewise, if dblog gets split to be on DDEV, then this would be best disabled on DDEV.

Reply:
Done